ssl/test/runner/runner.go

```diff
@@ -19692,8 +19704,41 @@ func addMultipleCertSlotTests() {
 				strictAlgTest.config.CipherSuites = []uint16{alg.cipher}
 			}
 
+			// Extend the original test, but force it to use the early certificate callback instead.
+			// Expected behavior should be the same.
+			strictAlgCallbackTest := strictAlgTest
+			strictAlgCallbackTest.name = prefix + "Multiple-Cert-Strict-Alg-Callback" + suffix
+			strictAlgCallbackTest.flags = append(strictAlgCallbackTest.flags, "-use-early-callback")
```

Comment:
What is this testing exactly? What happens when the callback and multiple certs are configured?

Reply:
This is testing when the certificates are set via the early certificate callback, rather than setting up everything in `SSL_CTX/SSL` before the handshake is initiated. The expected behavior should be the exact same, it's just setting the certs beforehand VS during the callback.